metrics/deploy/test/deploy/deploy_command_test.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:args/command_runner.dart';
+import 'package:deploy/deploy/deploy_command.dart';
+import 'package:test/test.dart';
+
+const _defaultUsage = '''
+Usage: metrics <command> [arguments]
+
+Global options:
+-h, --help    Print this usage information.
+
+Available commands:
+  help   Display help information for metrics.
+
+Run "metrics help <command>" for more information about a command.''';
+
+void main() {
```

Comment:
Can we move this field to the `main` function to avoid adding top-level variables? The same suggestion for the variable in the `doctor_command_test.dart` file.
```suggestion
void main() {
const _defaultUsage = '''
Usage: metrics <command> [arguments]

Global options:
-h, --help    Print this usage information.

Available commands:
  help   Display help information for metrics.

Run "metrics help <command>" for more information about a command.''';

```